src/Evolution/Systems/Cce/Initialize/NoIncomingRadiation.hpp

```diff
@@ -49,14 +49,14 @@ struct NoIncomingRadiation : InitializeJ {
         "Number of linearized inversion iterations."};
     static type lower_bound() noexcept { return 10; }
     static type upper_bound() noexcept { return 1000; }
-    static type default_value() noexcept { return 300; }
+    static type suggested_value() noexcept { return 300; }
   };
 
   struct RequireConvergence {
     using type = bool;
     static constexpr Options::String help = {
       "If true, initialization will error if it hits MaxIterations"};
-    static type default_value() noexcept { return false; }
+    static type suggested_value() noexcept { return false; }
```

Comment:
delete?

Reply:
I actually prefer keeping a suggested value for this one, but to make it `true` (when I initially wrote this, I anticipated that the typical desired behavior would be to proceed with the evolution, but in practice, users often do not want to continue when the initial data is known to be poorly matched with the worldtube.